gammapy/irf/effective_area.py

```diff
@@ -69,6 +69,25 @@ def plot(self, ax=None, energy=None, show_energy=None, **kwargs):
         return ax
 
     @classmethod
+    def from_parametrization(cls, energy, instrument='HESS'):
+        """Get parametrized effective area 
+
+        see :func:`~gammapy.irf.abramowski_effective_area`
```

Comment:
I think we should have a docs page or IPython notebook that just talks about loading data, IRFs and doing the quick-look print-out and peek plots.

Whether it's helpful to have plot directives that generate images as part of the API docs as well, I'm not sure.
I'm leaning slightly towards no, to try to prevent  the docs build from going slower and slower, plus when someone looks at API docs, they usually don't need to see plots.
But I'm almost 50:50 on this ... do what you like best!